diff --git a/index.ts b/index.ts
index 30581a23fe47f5ad5c03b37469472a738e6ed20d..b3bd69af152c2c662358dab29977a088f6515afe 100644
--- a/index.ts
+++ b/index.ts
@@ -2747,7 +2747,7 @@ export declare abstract class D1PreparedStatement {
 /**
  * A email message that is sent to a consumer Worker.
  */
-export interface EmailMessage<Body = unknown> {
+export interface EmailMessage {
   /**
    * Envelope From attribute of the email message.
    */