diff --git a/index.ts b/index.ts
index 67d6f819369e4c441ae1056c37cb6b352a901a83..3e32910356854b1737f5ca1e40de5301fa99d9a4 100644
--- a/index.ts
+++ b/index.ts
@@ -327,7 +327,7 @@ export type ExportedHandlerTestHandler<Env = unknown> = (
 ) => void | Promise<void>;
 export interface ExportedHandler<
   Env = unknown,
-  QueueMessage = unknown,
+  _ = unknown,
   CfHostMetadata = unknown
 > {
   fetch?: ExportedHandlerFetchHandler<Env, CfHostMetadata>;