diff --git a/dist/src/index.js b/dist/src/index.js
index c28f9dc8ca357561e27ea855bbb0dba8a0a10c4c..2d017c111a0b6fc656ef62a5fdb7f52b5bab0fb1 100644
--- a/dist/src/index.js
+++ b/dist/src/index.js
@@ -473,11 +473,7 @@ var D1Plugin = class extends import_shared.Plugin {
   async setup(storageFactory) {
     const bindings = {};
     for (const dbName of this.d1Databases ?? []) {
-      if (dbName.startsWith(D1_BETA_PREFIX)) {
-        bindings[dbName] = await this.getDatabase(storageFactory, dbName.slice(D1_BETA_PREFIX.length));
-      } else {
-        console.warn(`Not injecting D1 Database for '${dbName}' as this version of Miniflare only supports D1 beta bindings. Upgrade Wrangler and/or Miniflare and try again.`);
-      }
+      bindings[dbName] = await this.getDatabase(storageFactory, dbName);
     }
     return { bindings };
   }